quickstarts/maven/wildfly-jar/pom.xml

```diff
@@ -0,0 +1,146 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2020 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at:
+
+        https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.eclipse.jkube.quickstarts.maven</groupId>
+  <artifactId>wildfly-jar</artifactId>
+  <version>1.0.0-alpha-4</version>
+  <name>Eclipse JKube :: Quickstarts :: Maven :: Wildfly JAR</name>
+  <packaging>war</packaging>
+
+  <properties>
+    <version.wildfly.jar>2.0.0.Alpha6</version.wildfly.jar>
+    <version.wildfly>20.0.0.Final</version.wildfly>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <failOnMissingWebXml>false</failOnMissingWebXml>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <!-- crash image name -->
+    <!--<jkube.generator.name>%a/%g:%l</jkube.generator.name>-->
+  </properties>
+
+  <dependencies>
+    <dependency>
+        <groupId>org.jboss.spec.javax.ws.rs</groupId>
+        <artifactId>jboss-jaxrs-api_2.0_spec</artifactId>
+        <version>1.0.1.Final</version>
+        <scope>provided</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <finalName>wildfly-jar-sample</finalName>
+    <plugins>
+      <plugin>
+            <groupId>org.wildfly.plugins</groupId>
+            <artifactId>wildfly-jar-maven-plugin</artifactId>
+            <version>${version.wildfly.jar}</version>
+            <configuration>
+                <feature-pack-location>wildfly@maven(org.jboss.universe:community-universe)#${version.wildfly}</feature-pack-location>
+                <layers>
+                    <layer>cloud-server</layer>
+                </layers>
+                <excluded-layers>
+                    <layer>deployment-scanner</layer>
+                </excluded-layers>
+                <cloud/>
+            </configuration>
+            <executions>
+                <execution>
+                    <goals>
+                        <goal>package</goal>
+                    </goals>
+                </execution>
+            </executions>
+        </plugin>
+    </plugins>
+  </build>
+
+  <profiles>
+    <profile>
+      <id>kubernetes</id>
+      <build>
+	<plugins>
+	  <plugin>
+	    <groupId>org.eclipse.jkube</groupId>
+	    <artifactId>kubernetes-maven-plugin</artifactId>
+            <version>1.0.0-SNAPSHOT</version>
```

Comment:
Please bind this to ${project.version} to ease the maintenance process (same for line 117). 
Project version (line 23) can be set to `1.0.0-SNAPSHOT` since this won't work in current latest release (1.0.0-alpha-4).

Reply:
Ok, fixed.